cmd/root.go

```diff
@@ -7,7 +7,7 @@ import (
 	"log"
 	"os"
 
-	"github.com/sourcegraph/checkup"
+	"github.com/monosolutions/checkup"
```

Comment:
why is this changed?

Reply:
That is an error, mostly so i can use CheckUp with the changes i needed.